view/frontend/web/js/view/payment/method-renderer/cc.js

```diff
@@ -37,7 +37,9 @@ define([
                         contractOptions.push(item);
                     }
                 });
-                contractOptions.push({contractId:0,label:$t('Use Different Card')});
+                if (contractOptions.length){
+                    contractOptions.push({contractId:0,label:$t('Use Different Card')});
```

Comment:
Could we also put these in separate lines for readability?

ex.

```
{
     contractId : 0,
     label : $t('Use Different Card')
}
```

Reply:
```
                if (contractOptions.length){
                    contractOptions.push(
                        {
                            contractId:0,
                            label:$t('Use Different Card')
                        }
                        );
                }
```
? :)

Reply:
phpstorm wants it to look like this

Reply:
Really? If wont accept the following?

```
if (contractOptions.length){
     contractOptions.push( {
            contractId:0,
             label:$t('Use Different Card')
     });
 }
```